config.js

```diff
@@ -1,5 +1,5 @@
-noddityConfig = {
-	title: 'Noddity',
+window.noddityConfig = {
```

Comment:
Just for the possibility of running JavaScript with a DOM in non-browser environments (Electron or nwjs or whatever) I'd rather not reference `window` for globals